<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeakerDialog</class>
 <widget class="QDialog" name="SpeakerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loudspeaker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>

    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">

    <layout class="QFormLayout" name="speakerFormLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="vendorLabel">
       <property name="text">
        <string>Vendor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="vendorLineEdit">
       <property name="placeholderText">
        <string>speaker vendor name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="modelLabel">
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="modelLineEdit">
       <property name="placeholderText">
        <string>speaker name or reference number</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="fsLabel">
       <property name="text">
        <string>Fs (Hz)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="fsDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="qtsLabel">
       <property name="text">
        <string>Qts</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="qtsDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="vasLabel">
       <property name="text">
        <string>Vas (L)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="vasDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="diaLabel">
       <property name="text">
        <string>Dia cutout (m)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="diaDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>99.998999999999995</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="zLabel">
       <property name="text">
        <string>Z (Ohm)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="zDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="sdLabel">
       <property name="text">
        <string>Sd (mÂ²)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="sdDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="reLabel">
       <property name="text">
        <string>Re (Ohm)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="reDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="leLabel">
       <property name="text">
        <string>Le (mH)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="leDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="xmaxLabel">
       <property name="text">
        <string>Xmax (mm)</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="xmaxDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="qmsLabel">
       <property name="text">
        <string>Qms</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="qmsDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="qesLabel">
       <property name="text">
        <string>Qes</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="qesDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="splLabel">
       <property name="text">
        <string>Spl (dB)</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QDoubleSpinBox" name="splDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="peLabel">
       <property name="text">
        <string>Pe (W)</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QDoubleSpinBox" name="peDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="blLabel">
       <property name="text">
        <string>BL (Tm)</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QDoubleSpinBox" name="blDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="vcLabel">
       <property name="text">
        <string>Voice Coil (#)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="vcSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>

   </widget>
   </widget>

   </item>
   <item>
    <widget class="QDialogButtonBox" name="speakerButtonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>speakerButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpeakerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speakerButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpeakerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
